src/main/kotlin/uk/gov/dluhc/emsintegrationapi/service/AbstractApplicationService.kt

```diff
@@ -31,8 +31,8 @@ abstract class AbstractApplicationService(
         }
     }
 
-    protected fun sendMessage(request: EMSApplicationResponse, applicationId: String, queueNameOverride: QueueConfiguration.QueueName? = null) {
-        val targetQueueName = queueNameOverride ?: queueName
+    protected fun sendMessage(request: EMSApplicationResponse, applicationId: String, isFromApplicationsApi: Boolean? = null) {
+        val targetQueueName = isFromApplicationsApi?.takeIf { it }?.let { QueueConfiguration.QueueName.EMS_APPLICATION_PROCESSED_QUEUE } ?: queueName
```

Comment:
Sorry I found this quite hard to read 😅 Any chance we could do something like
```suggestion
        val targetQueueName = if (isFromApplicationsApi == true) QueueConfiguration.QueueName.EMS_APPLICATION_PROCESSED_QUEUE else queueName
```